src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReleaseServiceTests.cs

```diff
@@ -332,9 +332,9 @@ await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContex
 
             var dataBlockService = new Mock<IDataBlockService>(MockBehavior.Strict);
             var dataImportService = new Mock<IDataImportService>(MockBehavior.Strict);
-            var subjectService = new Mock<ISubjectService>(MockBehavior.Strict);
+            var subjectService = new Mock<ISubjectRepository>(MockBehavior.Strict);
```

Comment:
There's some cases throughout this class where the variables e.g. `subjectService` and `releaseSubjectService` haven't be renamed. I think it's been done consistently elsewhere in this PR so I think we should rename them here too?

Reply:
Good point! I looked through all of the other usages (or at least most of them) and renamed all the variables I could find. I'm sure people will be diligent and change any residual instances they encounter in the future :smile: